### PR TITLE
Add Kafka 2.6.0 to tests and protocol compatibility matrix

### PR DESCRIPTION
**Pull request by methodmissing**
References https://downloads.apache.org/kafka/2.6.0/RELEASE_NOTES.html

---
This change is [Reviewable](https://reviewable.io/reviews/dpkp/kafka-python/2162)

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 kafka-python
 ############
 
-.. image:: https://img.shields.io/badge/kafka-2.5%2C%202.4%2C%202.3%2C%202.2%2C%202.1%2C%202.0%2C%201.1%2C%201.0%2C%200.11%2C%200.10%2C%200.9%2C%200.8-brightgreen.svg
+.. image:: https://img.shields.io/badge/kafka-2.6%2C%202.5%2C%202.4%2C%202.3%2C%202.2%2C%202.1%2C%202.0%2C%201.1%2C%201.0%2C%200.11%2C%200.10%2C%200.9%2C%200.8-brightgreen.svg
     :target: https://kafka-python.readthedocs.io/compatibility.html
 .. image:: https://img.shields.io/pypi/pyversions/kafka-python.svg
     :target: https://pypi.python.org/pypi/kafka-python
```
Comment (vvuibert) [inline docs/index.rst:7]: https://github.com/dpkp/kafka-python/blob/4d65e90aabf3d4ede355d76ad9ca62e64f7fd3c9/docs/index.rst#protocol missed updating 2.4+ to 2.6+

Comment (methodmissing) [reply docs/index.rst:7]: Ready to push but will do so when build completes

**Comment by dpkp** [comment]
This is great, thank you!

File: servers/2.6.0/resources/kafka_server_jaas.conf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+KafkaServer {{
+    {jaas_config}
+}};
+Client {{}};
```
Comment (gabriel-tincu) [inline servers/2.6.0/resources/kafka_server_jaas.conf:4]: newline needed here?

**Comment by dpkp** [comment]
Thanks!!

**Pull request merged**